app/components/header.tsx

```diff
@@ -1,20 +1,9 @@
 import React from "react";
 
 /**
```

Comment:
The floating is no longer defined here. Now, this is a a very generic header container. Could have just left a <header> tag in layout.tsx and removed this entirely, but i prefer having a component.